src/Executables/Atlas/3. Configuration/1. General Configuration/Start Menu/Install Open-Shell.cmd

```diff
@@ -6,10 +6,16 @@ whoami /user | find /i "S-1-5-18" > nul 2>&1 || (
 	exit /b
 )
 
-curl -L --output !windir!\AtlasModules\Open-Shell.exe https://github.com/Open-Shell/Open-Shell-Menu/releases/download/v4.4.189/OpenShellSetup_4_4_189.exe
-if exist "!windir!\SystemApps\Microsoft.Windows.Search_cw5n1h2txyewy" goto existOS
-if exist "!windir!\SystemApps\Microsoft.Windows.StartMenuExperienceHost_cw5n1h2txyewy" goto existOS
-goto rmSSOS
+ping -n 1 -4 1.1.1.1 | find "time=" > nul 2>&1
```

Comment:
Cloudflare is blocked in some places, maybe using something like https://example.com/ is better?